sklearn/svm/base.py

```diff
@@ -484,6 +484,17 @@ def coef_(self):
     def _get_coef(self):
         return safe_sparse_dot(self._dual_coef_, self.support_vectors_)
 
+    @property
+    def n_support_(self):
+        check_is_fitted(self)
```

Comment:
I think a `@property` raising a `NotFittedError` is kinda odd. Rather have an `AttributeError` here.